master/src/main/scala/org/apache/celeborn/service/deploy/master/MasterArguments.scala

```diff
@@ -44,7 +44,9 @@ class MasterArguments(args: Array[String], conf: CelebornConf) {
     val localNode = clusterInfo.localNode
     _host = _host.orElse(Some(conf.haMasterNodeHost(localNode.nodeId)))
     _port = _port.orElse(Some(conf.haMasterNodePort(localNode.nodeId)))
-    _internalPort = _internalPort.orElse(Some(conf.haMasterNodeInternalPort(localNode.nodeId)))
+    _internalPort = _internalPort.orElse(if (conf.internalPortEnabled)
+      Some(conf.haMasterNodeInternalPort(localNode.nodeId))
+    else None)
```

Comment:
is the following style allowed by our scalafmt rules?
```suggestion
    _internalPort = _internalPort.orElse { 
      if (conf.internalPortEnabled) Some(conf.haMasterNodeInternalPort(localNode.nodeId)) else None
    }
```